pkg/e2e/ddev_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/icmd"
+)
+
+const ddevVersion = "v1.18.2"
+
+func TestComposeRunDdev(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Running on Windows. Skipping...")
+	}
+	c := NewParallelE2eCLI(t, binDir)
+	dir, err := os.MkdirTemp("", t.Name()+"-")
+	if err != nil {
+		return
+	}
```

Comment:
Should this not be
```suggestion
	assert.NotNil(err)
```
(or something like that, without completion I’m never sure which it is with the different testing libraries we use 😬)

Reply:
Sure! I think that was my automatic error checking kicking in!
Good catch!